.github/labeler.yml

```diff
@@ -21,3 +21,15 @@ github_actions:
       - changed-files:
           - any-glob-to-any-file:
               - .github/**/*
+
+java:
+  - any:
+      - changed-files:
+          - any-glob-to-any-file:
+              - '**/*.java'
+
+tests:
+  - any:
+      - changed-files:
+          - any-glob-to-any-file:
+              - integration-tests/**/*
```

Comment:
This isn't enough.
I think it should be src/test/java and src/test/groovy instead